hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNode.java

```diff
@@ -2671,6 +2676,18 @@ private String reconfigureDecommissionBackoffMonitorParameters(
     }
   }
 
+  private String reconfigureMinBlocksForWrite(String property, String newValue)
+      throws ReconfigurationException {
+    try {
+      int newSetting = adjustNewVal(
+          DFS_NAMENODE_BLOCKPLACEMENTPOLICY_MIN_BLOCKS_FOR_WRITE_DEFAULT, newValue);
+      this.namesystem.getBlockManager().setMinBlocksForWrite(newSetting);
```

Comment:
``this.namesystem`` isn't required, can be just ``namesystem.getBlockManager()``?

Reply:
Thanks Sir @ayushtkn review and suggestion.
I will update it later.